.github/workflows/nix-shell-build.yml

```
@@ -11,8 +11,8 @@ jobs:
   build:
     strategy:
       matrix:
-        nixos-version: [ "nixos-24.05", "nixos-unstable" ] 
-        bitcoin-ref: [ "master", "v27.1" ]
+        nixos-version: [ "nixos-24.05", "nixos-unstable" ]
```

feel free to update to 24.11 too
```suggestion
        nixos-version: [ "nixos-24.11", "nixos-unstable" ]
```